diff --git a/src/SdFatConfig.h b/src/SdFatConfig.h
index 4b64f9d..eacf623 100644
--- a/src/SdFatConfig.h
+++ b/src/SdFatConfig.h
@@ -166,7 +166,7 @@
  * 3 - An external SPI driver derived from SdSpiBaseClass is always used.
  */
 #ifndef SPI_DRIVER_SELECT
-#define SPI_DRIVER_SELECT 0
+#define SPI_DRIVER_SELECT 2
 #endif  // SPI_DRIVER_SELECT
 /**
  * If USE_SPI_ARRAY_TRANSFER is one and the standard SPI library is
